### DISCUSSION TITLE
Feature Question: Tampermonkey/tamperdav

### DISCUSSION THREAD

**Discussion by Malix-Labs** [lisonge/vite-plugin-monkey#65] [Q&A] [answered]
Did you thought about using [Tampermonkey/tamperdav](https://github.com/Tampermonkey/tamperdav)?

**Comment by lisonge** [comment, marked as answer]
no

**Reply by Malix-Labs** [reply]
Do you think it would be a good idea to implement it?

**Reply by lisonge** [reply]
> Do you think it would be a good idea to implement it?

sorry, I have no interest in it

**Reply by Malix-Labs** [reply]
Okay, no problem.
I honestly don't even know if that would be good.
